cmd/influxd/inspect/dump_wal/dump_wal.go

```diff
@@ -43,20 +43,20 @@ It has two modes of operation, depending on the --find-duplicates flag.
 	return cmd
 }
 
-func (dumpWAL *dumpWALCommand) run(args []string) error {
+func (dumpWAL *dumpWALCommand) run(cmd cobra.Command, args []string) error {
```

Comment:
I think it might make sense to keep this as passing a pointer around instead of direct object

Reply:
That's fair - I'll go ahead and change to a pointer for those.